.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -1,14 +1,62 @@
 ---
 name: Bug Report
-about: Report 🐞 Bugs
-title: ''
+about: Create a 🐞 Bug Report to Improve Athens
+title: 'Descriptive Title with Category'
 labels: 'type: 🐞 bug'
 assignees: ''
+ID number: ''
+Category: ''
 
 ---
 
-**Problem**
+**Before Beginning:**
+
+- [ ] Read [Contribution Guidelines](https://app.gitbook.com/@athensresearch/s/handbook/~/drafts/-MZt8Vmd14cTrIp7xfV9/contributing-1/code-of-conduct/@archived) and [Code of Conduct](https://app.gitbook.com/@athensresearch/s/handbook/~/drafts/-MZt8Vmd14cTrIp7xfV9/contributing-1/code-of-conduct/@archived)
+
+- [ ] Check for similar [issues](https://github.com/athensresearch/athens/issues)
+- [ ]  Ask questions in [Discussions](https://github.com/athensresearch/athens/discussions)  *Bug reports ≠ questions!*
```

Comment:
nit: extra space after `[ ]`

Reply:
nit: maybe explicitly say what to do if there's similar issues, and to not use issues to ask questions?

e.g.:

"Check for similar [issues](...). If you find one, [... upvote it / comment on it with your details / ... ] instead of creating a new one." (maybe also: "This helps us developers in ways X and Y.")

"Is this a question, not an issue? Then ask in [Discussion](...)"